package.json

```diff
@@ -11,31 +11,30 @@
   },
   "license": "MIT",
   "dependencies": {
-    "chalk": "2.4.2",
-    "chokidar": "2.1.5",
-    "compression": "1.7.x",
-    "express": "4.16.4",
-    "fs-extra": "7.0.1",
-    "highlight.js": "9.15.6",
-    "lodash": "4.17.11",
-    "marked": "0.6.2",
-    "mkdirp": "0.5.1",
-    "q": "1.5.1",
-    "slug": "1.1.0",
+    "chalk": "^2.4.2",
```

Comment:
Last serious bug in highlight.js showed us that versions of libs should be fixed and updated only after all tests are passed. I think it's the best for stability of projects that use md-docs.

Reply:
Ok